pkg/option/config.go

```diff
@@ -639,6 +639,9 @@ const (
 	// EnableLocalNodeRoute controls installation of the route which points
 	// the allocation prefix of the local node.
 	EnableLocalNodeRoute = "enable-local-node-route"
+
+	// EnableRemoteNodeIdentity enables use of the non-local-node identity
+	EnableRemoteNodeIdentity = "enable-non-local-node-identity"
```

Comment:
I notice there's still some `non-local` references here, in the docs, and one or two other places like code comments. Did you intend to also change these?

Reply:
Yes, I've already changed it. I will push after I'm done adding the CI tests.